### ISSUE TITLE
multi user advice

### ISSUE THREAD

**Issue by jackzzz** [MASGAU/MASGAU#82]
This is really a good software!!!
But will you please consider about add on multi user with password protect? Because my friend always overwrite my game record.

Thank you very much!

**Comment by OranguTech** [comment]
Then don't let your "friend" log in to your account - create one for him.

**Comment by sanmadjack** [comment]
I have to admit, Scandalon is right about this. This is why Windows has shipped with a "Guest" account for over 10 years. How is it that you imagine this feature working? Can you step me through how it would be used?

**Comment by jackzzz** [comment]
ok I just give up.

**Comment by sanmadjack** [comment]
No, please don't give up I'm just trying to understand how it is that you see this being implemented. I think I'm not completely understanding what you're describing.

**Comment by jackzzz** [comment]
ok,thanks, the computer is being used in dorm. So i just want a single software  plan to handle that problem. Actually for now I am using the script to backup the saves for different friend. So do worry about that if its too complex for u. Thanks a lot. :)
